### ISSUE TITLE
[Question] Disabling Individual Perks

### ISSUE THREAD

**Issue by greenking13** [HellFirePvP/AstralSorcery#428]
Simple question, is it possible to disable the perks given when one aligns themselves to a constellation? I'm currently aligned to Aevitas at the max level and wanted to disable the mineral perk, since I am converting my entire floor into ore. I don't mean disable period, but more of a toggle, so that I could only convert stone into ore when I'm within my mines or etc.

**Comment by HellFirePvP** [comment]
Nope. Perks cannot be disabled unless you un-attune yourself and yes, it has been suggested many times before. (In your circumstance, this might also be a good push towards actually building & designing a base that is more than just cleanstone floor, walls and ceiling)

**Comment by greenking13** [comment]
Very well, I primarily asked because a base mate of mine was annoyed about it. About your parenthesized  statement, we decided to use smooth stone intentionally prior to being aware of your ore perk, it's not that we just built out of solely smooth stone, (As shown when I stated my floor and not the building.) but that we decided that smooth stone went somewhat well with his vision of a "modern" house. (Don't question it, I don't even understand how myself.)

As for another question, after looking through the configs, I don't see a method of reducing the ore conversion rate, besides possible the ritual potency, which just from its name alone, makes me doubt that it affects the passive attunement perks. Since besides that, I don't see a simple method of reducing the rate besides the same base mate's suggestion of introducing smooth stone as a possible conversion. (Which I think is just silly, but hey, he's the second in charge for the pack's development.)
